ArduCopter/RC_Channel.h

```diff
@@ -46,6 +46,17 @@ class RC_Channels_Copter : public RC_Channels
     // returns true if throttle arming checks should be run
     bool arming_check_throttle() const override;
 
+    // support for trimming AHRS using RC stick inputs:
```

Comment:
need to update the comment?  perhaps just replace "RC stick inputs:" with "an auxiliary function"